website/src/_posts/2018-10-29-beam-2.8.0.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title:  "Apache Beam 2.8.0"
+date:   2018-10-29 00:00:01 -0800
+excerpt_separator: <!--more-->
+categories: blog
+authors:
+        - altay
+
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+We are happy to present the new 2.8.0 release of Beam. This release includes both improvements and new functionality.
+See the [download page]({{ site.baseurl }}/get-started/downloads/#280-2018-10-26) for this release.<!--more-->
+For more information on changes in 2.8.0, check out the
+[detailed release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12343985).
+
+## New Features / Improvements
+
+### Dependency Upgrades
+
+* Elastic Search dependency upgraded to 6.3.2
+* google-cloud-pubsub dependency upgraded to 0.35.4
+* google-api-client dependency upgraded to 1.24.1
+* Updated Flink Runner to 1.5.3
+* Updated Spark runner to Spark version 2.3.2
+
+### SDKs
+
+* Python SDK added support for user state and timers.
+* Go SDK added support for side inputs.
+
+### Portability
+
+* [Python on Flink MVP](https://beam.apache.org/roadmap/portability/#python-on-flink)
+* Added support for side Inputs over the Fn API.
```

Comment:
Is this meaningful to users? If yes, clarify what it means else remove it?

Reply:
Fair point. I will remove it. (I got this from https://issues.apache.org/jira/browse/BEAM-2863, I am not sure why it has a fix version 2.8.0.)